settings.sh.sample

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
This line can go away, no shebang needed when sourcing.

In case this file's executable bit is set, please change file mode to 0644 (didn't check myself)